src/containers/ExpandedPostModal.js

```diff
@@ -10,19 +10,28 @@ import Post from '../components/Post'
 export default class ExpandedPostModal extends React.Component {
   static propTypes = {
     dispatch: func,
+    socket: object,
     commentId: string,
     post: object
   }
 
   componentDidMount () {
-    const { commentId, post: { id }, dispatch } = this.props
+    const { commentId, post: { id, numComments }, dispatch, socket } = this.props
     dispatch(fetchComments(id, {offset: 3}))
     .then(() => {
       if (commentId) {
         const wrapper = document.getElementById(modalWrapperCSSId)
         scrollToComment(commentId, wrapper)
       }
     })
+    socket.on(`p/${id}/comment_added`, function (){
+      dispatch(fetchComments(id, {offset: numComments, refresh: true}))
```

Comment:
So, to be clear, are you saying to still perform an API call after receiving the socket event, but be sure to only fetch the comment that was just created? Or are you saying that it would be good to pass the comment object down with the websocket event? :)